cpp/oneapi/dal/io/detail/load_graph.hpp

```diff
@@ -19,20 +19,24 @@
 #include <algorithm>
 #include <fstream>
 
-#include "oneapi/dal/detail/common.hpp"
+#include "oneapi/dal/common.hpp"
+#include "oneapi/dal/detail/threading.hpp"
 #include "oneapi/dal/exceptions.hpp"
-#include "oneapi/dal/graph/detail/graph_container.hpp"
-#include "oneapi/dal/graph/detail/undirected_adjacency_array_graph_impl.hpp"
-#include "oneapi/dal/graph/graph_common.hpp"
-#include "oneapi/dal/graph/undirected_adjacency_array_graph.hpp"
+#include "oneapi/dal/graph/common.hpp"
+#include "oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.hpp"
+#include "oneapi/dal/graph/undirected_adjacency_vector_graph.hpp"
 #include "oneapi/dal/io/detail/load_graph_service.hpp"
-#include "oneapi/dal/detail/threading.hpp"
 #include "oneapi/dal/io/graph_csv_data_source.hpp"
 #include "oneapi/dal/io/load_graph_descriptor.hpp"
 #include "services/daal_atomic_int.h"
 #include "services/daal_memory.h"
```

Comment:
We have an agreement to not include DAAL header files to oneAPI header files. I understand `daal_atomic_int`, but `daal_memory` functionality is fully available in oneAPI part. 

Could `daal_atomic_int` be replaced?

Reply:
will be done ww7-8